components/micro-gateway-core/src/main/ballerina/gateway/constants/constants.bal

```diff
@@ -343,6 +343,8 @@
 @final string KEY_OAUTH_PROVIDER = "OAuthAuthProvider";
 @final string KEY_UPLOAD_TASK = "UploadTimerTask";
 @final string KEY_ROTATE_TASK = "RotateTimerTask";
+@final string KEY_ETCD_UTIL = "EtcdUtil";
 
 
 @final public int DEFAULT_LISTENER_TIMEOUT = 120000; //2 mins
+@final public int DEFAULT_ETCD_TRIGGER_TIME = 10000; //10 seconds
```

Comment:
Can we increase the default value, 10 seconds peiodic task seems too freqeunt. Is it fine to increase this value

Reply:
Yes we can increase this value. Any suggestions?
Also this value is overridable.

Reply:
Lets make the default value 60 seconds